core/src/test/java/org/apache/iceberg/TestMetadataUpdateParser.java

```diff
@@ -1240,10 +1234,10 @@ private static void assertEqualsAddSnapshot(
   }
 
   private static void assertEqualsRemoveSnapshot(
```

Comment:
this method can be removed, since there's already one that takes `RemoveSnapshots`